nistats/tests/test_verify_events_file_uses_tab_separators.py

```diff
@@ -0,0 +1,137 @@
+import csv
+import os
+from tempfile import NamedTemporaryFile
+
+import pandas as pd
+from nose.tools import (assert_raises,
+                        assert_true,
+                        )
+
+from nistats.utils import _verify_events_file_uses_tab_separators
+
+
+def make_data_for_test_runs():
+    data_for_temp_datafile = [
+        ['csf', 'constant', 'linearTrend', 'wm'],
+        [13343.032102491035, 1.0, 0.0, 9486.199545677482],
+        [13329.224068063204, 1.0, 1.0, 9497.003324892803],
+        [13291.755627241291, 1.0, 2.0, 9484.012965365506],
+        ]
+    
+    delimiters = {
+        'tab': '\t',
+        'comma': ',',
+        'space': ' ',
+        'semicolon': ';',
+        'hyphen': '-',
+        }
+    
+    return data_for_temp_datafile, delimiters
+
+
+def _create_test_file(temp_csv, test_data, delimiter):
+    csv_writer = csv.writer(temp_csv, delimiter=delimiter)
+    for row in test_data:
+        csv_writer.writerow(row)
+    temp_csv.flush()
+
+
+def _run_test_for_invalid_separator(filepath, delimiter_name):
+    if delimiter_name not in ('tab', 'comma'):
+        with assert_raises(ValueError):
+            _verify_events_file_uses_tab_separators(events_files=filepath)
+    else:
+        result = _verify_events_file_uses_tab_separators(events_files=filepath)
+        assert_true(result is None)
```

Comment:
Then this should always be the case.

Reply:
Sorry. 
results is alway None, whether the file is valid or None.

Reply:
The verification function raises a ValueError if the file is a text file with valid path BUT does not use tabs or commas.
Every other case, the verification function returns None. Other errors are caught by other functions in the original code.
Here the comment is about the unit test I wrote for the verification function.

Reply:
All right, you're too subtle for me.